### PR TITLE
Updated timeline molecule and atom, added one more state as failure

### PR DESCRIPTION
**Pull request by rachana-egov**

### CODE REVIEW

File: flutter/digit-ui-components/digit_components/lib/enum/app_enums.dart [hunk 1]
```diff
@@ -65,7 +65,7 @@ enum DropdownType {
 enum InfoType { info, success, error, warning }
 
 /// Enum to define different state for timeline
-enum TimelineStepState { completed, present, future }
+enum TimelineStepState { completed, present, future, failed }
```
Comment (naveen-egov) [inline flutter/digit-ui-components/digit_components/lib/enum/app_enums.dart:68]: Change the enum to uppercase

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/timeline_stories.dart [hunk 2]
```diff
@@ -8,104 +8,165 @@ import 'package:storybook_toolkit/storybook_toolkit.dart';
...
-            value: TimelineStepState.completed,
-          ),
-          Option(
-            label: 'Future',
-            value: TimelineStepState.future,
+      name: 'Atom/Timeline/Ongoing',
+      builder: (context) {
+
+        bool showAdditionalWidget = context.knobs.boolean(
+          label: "Show Additional Widget",
```
Comment (naveen-egov) [inline flutter/digit-ui-components/storybook/lib/widgets/atoms/timeline_stories.dart:15]: use i18 codes

**Review by naveen-egov — CHANGES_REQUESTED** [review]
please make the changes requested

File: flutter/digit-ui-components/storybook/lib/widgets/atoms/timeline_stories.dart (hunk 2 above)
Comment (rachana-egov) [reply flutter/digit-ui-components/storybook/lib/widgets/atoms/timeline_stories.dart:15]: this is storybook not component

File: flutter/digit-ui-components/digit_components/lib/enum/app_enums.dart (hunk 1 above)
Comment (rachana-egov) [reply flutter/digit-ui-components/digit_components/lib/enum/app_enums.dart:68]: enum name is pascal case only and enum value in camel case

**Review by naveen-egov — APPROVED** [review]

**Pull request merged**